stubs/psutil/psutil/_common.pyi

```diff
@@ -244,13 +246,13 @@ _Func = TypeVar("_Func", bound=Callable[..., Any])
 def usage_percent(used, total, round_: int | None = ...) -> float: ...
 def memoize(fun: _Func) -> _Func: ...
 def memoize_when_activated(fun: _Func) -> _Func: ...
-def isfile_strict(path) -> bool: ...
-def path_exists_strict(path) -> bool: ...
+def isfile_strict(path: StrOrBytesPath) -> bool: ...
+def path_exists_strict(path: StrOrBytesPath) -> bool: ...
 def supports_ipv6() -> bool: ...
 def parse_environ_block(data): ...
-def sockfam_to_enum(num): ...
-def socktype_to_enum(num): ...
-def conn_to_ntuple(fd, fam, type_, laddr, raddr, status, status_map, pid: Any | None = ...): ...
+def sockfam_to_enum(num: int) -> AddressFamily: ...
+def socktype_to_enum(num: int) -> SocketKind: ...
+def conn_to_ntuple(fd: int, fam: int, type_: int, laddr: str, raddr: str, status: str, status_map, pid: int | None = ...): ...
```

Comment:
`laddr` and `raddr` are not strings when the family is `AF_INET` or `AF_INET6`. The implementation is:

```python
def conn_to_ntuple(fd, fam, type_, laddr, raddr, status, status_map, pid=None):
    """Convert a raw connection tuple to a proper ntuple."""
    if fam in (socket.AF_INET, AF_INET6):
        if laddr:
            laddr = addr(*laddr)
        if raddr:
            raddr = addr(*raddr)
    ...
    if pid is None:
        return pconn(fd, fam, type_, laddr, raddr, status)
    else:
        return sconn(fd, fam, type_, laddr, raddr, status, pid)
```

Because this function is only in `psutil._common` and not exposed in `psutil`, it might be best to leave it unannotated.

Reply:
Yes, good catch. I will remove annotation for `laddr` and `raddr`